adapters/src/test/resources/h2-driver-datasource-ds.xml

```diff
@@ -12,6 +12,7 @@
   </datasource>
   <drivers>
       <driver name="h2">
+         <driver-class>org.h2.Driver</driver-class>
```

Comment:
Provides both driver-class and datasource-class, but no connection_url, it should work.